README.md

```diff
@@ -129,6 +130,22 @@ Options:
                         variety to lower percentages
   -p FILTER_PATHS, --path=FILTER_PATHS
                         Filter all modules not in a specified path
+  --compare             Compare two graphs with identical structure.
```

Comment:
This portion of `README.md` should be the unmodified output of `gprof2dot.py --help`.

Perhaps it would make sense to rename all compare options as `--compare-xxx`.